mmv1/products/bigqueryconnection/Connection.yaml

```diff
@@ -125,6 +125,27 @@ examples:
     primary_resource_id: "connection"
     vars:
       connection_id: "my-connection"
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'bigquery_connection_kms'
+    pull_external: true
+    primary_resource_id:
+      'bq-connection-cmek'
+    vars:
+      database_instance_name: 'my-database-instance'
+      username: 'user'
+      deletion_protection: 'true'
+      kms_key_name: 'bq-key'
```

Comment:
This should be a full name since it's not a reference - `projects/[kms_project_id]/locations/[region]/keyRings/[key_region]/cryptoKeys/[key]`

Reply:
Looked at the other example. Hope this works...